### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Yoanf26**
Update french language

### CODE REVIEW

**Comment by Yoanf26** [comment]
Add Roomba 976 to README

File: README.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ If this card works with your vacuum cleaner, please open a PR and your model to
 - **Roborock** S7, S6 (MaxV, Pure), S5 (Max), S50, S4 (Max), E25, E4
 - **Mijia** Robot Vacuum Cleaner 1C (STYTJ01ZHM)
 - **Xiaomi** Mi Robot (STYJ02YM), Mi Robot 1S, Mi Roborock V1 (SDJQR02RR), Mijia 1C, Mi Robot Vacuum-Mop P
-- **Roomba** 670, 675, 676, 960980, 981, i3, i7+, e5, S9, s9+, j7
+- **Roomba** 670, 675, 676, 960, 976, 980, 981, i3, i7+, e5, S9, s9+, j7
```
Comment (denysdovhan) [inline README.md:248]: This should be in a separate PR.

File: src/translations/fr.json [hunk 2]
```diff
@@ -13,7 +13,15 @@
     "returning": "Retour à la base",
     "docked": "A la base",
     "unknown": "Inconnu",
-    "offline": "Déconnecté"
+    "offline": "Déconnecté",
+    "Stopped": "Arrêté",
```
Comment (denysdovhan) [inline src/translations/fr.json:17]: These statuses seem like something very custom. I'll implement a better solution for that.

**Review by denysdovhan — COMMENTED** [review]
Closing this since I have a better solution for that.
